src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/GroupChat.java

```diff
@@ -40,4 +42,14 @@ public String getName() {
     public void setName(String name) {
         this.name = name;
     }
+
+    @Override
+    public String getHumanReadableNameForReceiver(User sender) {
+        if (getName() == null || getName().isBlank()) {
+            return getConversationParticipants().stream().map((participant) -> participant.getUser().getName()).collect(Collectors.joining(", "));
```

Comment:
This might get quite long with 5+ users. Will this be truncated in the client?

Reply:
Good suggestion. I added a limit of 100 characters which the string will be trimmed to. Also I add a "…" character when we trim so that the user knows the string is not complete.